### PR TITLE
RFP: Host test-suite

### PR DESCRIPTION
**Pull request by Noc2**
Create and maintain a comprehensive test-suite for conformance of core functionalities of Polkadot Host

### CODE REVIEW

File: docs/RFPs/Open/polkadot-protocol_conformance_tests.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- parachain candidate validation (once there are sufficient implementations that support parachains)
+
+The goal of the initial grant should be to develop a PoC. The long-term goal should be to develop a comprehensive test suite that is funded by the on-chain treasury. 
+
+**Useful resources:**
+- [Polkadot Protocol Specification](https://spec.polkadot.network/)
+- [GitHub polkadot-tests](https://github.com/w3f/polkadot-tests)
+- [Chopsticks](https://github.com/AcalaNetwork/chopsticks)
+- [Zombienet](https://github.com/paritytech/zombienet) 
+
```
Comment (bhargavbh) [inline docs/RFPs/Open/polkadot-protocol_conformance_tests.md:42]: [try-runtime](https://docs.substrate.io/reference/command-line-tools/try-runtime/)

File: docs/RFPs/Open/polkadot-protocol_conformance_tests.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        - block validation
+        - next/current validators aka VRF/block lottery
+        - secondary slot verification
+    - GRANDPA
+        - block import
+        - block validation
+        - justification import & validation/verification
+- trie proof verification
+- scale encoding and decoding, for specific message types, and randomly generated ones
+- parachain candidate validation (once there are sufficient implementations that support parachains)
```
Comment (bhargavbh) [inline docs/RFPs/Open/polkadot-protocol_conformance_tests.md:33]: we can remove "(once there are sufficient implementations that support parachains)". It made sense only in the context of differential testing.

**Review by bhargavbh — APPROVED** [review]
LGTM. added link to try-runtime and removed a not-so-relevant note. we can publish

File: docs/RFPs/Open/polkadot-protocol_conformance_tests.md (hunk 1 above)
Comment (Noc2) [reply docs/RFPs/Open/polkadot-protocol_conformance_tests.md:43]: Done

File: docs/RFPs/Open/polkadot-protocol_conformance_tests.md (hunk 2 above)
Comment (Noc2) [reply docs/RFPs/Open/polkadot-protocol_conformance_tests.md]: Removed. Thanks

**Review by semuelle — APPROVED** [review]

**Review by dsm-w3f — APPROVED** [review]

**Review by SBalaguer — APPROVED** [review]

**Pull request merged**